tests/acceptance/blocks/pagination.js

```diff
@@ -14,6 +15,7 @@ class Pagination {
    */
   async nextResults () {
     await Page.scrollToBottom(); // We must scroll to the bottom of the page to be able to click the next results page button
```

Comment:
For my own edification, we first need to scroll to the bottom of the page, then scroll to the bottom of the VR to click the button?

Reply:
Correct. It's needed primarily for screen with smaller heights. I'm actually making an item for this because when the height of the browser is small, the vertical results scroll bar can get hidden, and scrolling the page as well is the only way to get to the bottom of the vertical results